ci/prow/e2e-crd-variable

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Deploy KMMO..."
+make deploy
+
+echo "Wait until the KMMO Deployment is Available and "
+echo "Describe the Deployment and get its YAML if that failed..."
+timeout 1m kubectl wait --for condition=Available deployments.apps -n kmm-operator-system kmm-operator-controller-manager || kubectl describe deployments.apps -n kmm-operator-system kmm-operator-controller-manager && kubectl get -o yaml deployments.apps -n kmm-operator-system kmm-operator-controller-manager
```

Comment:
How about having something like that:
```
#!/usr/bin/env bash
on_error() { 
   echo "Some error occurred"
}

trap 'on_error' ERR

ls ~/dir_not_exists
```
See ref: https://tecadmin.net/execute-specific-command-on-error-in-bash-script/

Reply:
IMHO it is simpler to use an operand in this case. If it succeeds stop and if it does not succeed then show whatever.

Reply:
But then you need to repeat that for every command that may fail. Isn't it?

Reply:
Let's pull all the troubleshooting in one step called by the Action YAML?